app/code/Magento/Variable/README.md

```diff
@@ -1 +1,39 @@
-Magento\Variable Allows to create custom variables and then use them in email templates or in WYSIWYG editor for editing description of system entities.
+# Magento_Variable module
+
+The **Magento_Variable** module allows creating custom variables and then use them in email templates or in WYSIWYG editor for editing description of system entities.
+
+## Installation details
+
+Before installing this module, note that the **Magento_Variable** is dependent on the following modules:
+
+- Magento_Config
+- Magento_Store
+
+Before disabling or uninstalling this module, please consider the following dependencies:
+
+- Magento_Cms
+- Magento_Email
+
+Please find here [how to enable or disable modules in Magento 2](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-enable.html).
```

Comment:
Change all these "Please find here..." to "Refer to [...) for more information."